src/react-components/src/components/01-atoms/Button/Button.stories.tsx

```diff
@@ -42,4 +42,38 @@ export const buttonIconChildrenElement = () =>
   >
     <span>I'm wrapped in a span!</span>
   </Button>
-;
\ No newline at end of file
+  ;
+export const buttonIconRightLarge = () =>
+  <Button
+    callback={action("clicked")}
+    id="button"
+    type="outline"
+    iconPosition="right"
+    modifiers={["large"]}
+    iconName="plus"
+    iconDecorative={true}
+  >
+    <span>button content</span>
+  </Button>;
+
+export const linkLikeButton = () =>
+  <Button
+    callback={action["clicked"]}
+    id="button"
+    type="link"
+  >
+    <span>button content</span>
+  </Button >;
+
+export const linkLikeButtonIconLeft = () =>
+  <Button
+    callback={action["clicked"]}
+    id="button"
+    type="link"
+    iconPosition="left"
+    iconName="arrow-xsmall"
+    iconModifiers={["left"]}
+    iconDecorative={true}
+  >
+    <span>button content</span>
+  </Button >;
```

Comment:
Should we add a story for a `submit` button type, just to show how that works? The button used in `DateRangeForm` should be one, no?

Reply:
In stories, it shouldn't be different.  However, I will add buttonType to the stories to show that they can be passed.